src/sentry/api/bases/organization.py

```diff
@@ -45,7 +45,7 @@ def has_object_permission(self, request, view, organization):
             )
 
         elif request.auth:
-            return request.auth.organization_id == organization.id
```

Comment:
this isnt heavily used yet but its going to move us towards using a single source for 'what memberships and permissions are available to me'